packages/new-polymath-issuer/src/pages/Home.tsx

```diff
@@ -90,12 +92,24 @@ export class ContainerBase extends Component<Props> {
         name: 'My Dividend Distribution',
         amount: new BigNumber('10000'),
         checkpointId: 1,
-        excludedAddresses: [],
+        excludedAddresses: ['0x821aea9a577a9b44299b9c15c88cf3087f3b5544'],
+        // excludedAddresses: [],
         pushPaymentsWhenComplete: true,
       })
     );
   };
 
+  public startUpdateTaxWithholdings = () => {
+    this.props.dispatch(
+      updateTaxWithholdingListStart({
+        securityTokenSymbol: 'A0T0',
+        dividendType: DividendModuleTypes.Erc20,
+        investorAddresses: ['0x0d1d4e623d10f9fba5db95830f7d3839406c6af2'],
+        percentages: [0.5],
+      })
+    );
+  };
+
   public checkForValidity = async () => {
```

Comment:
This log is from before this PR apparently, but lets remove it

Reply:
Same as above